reactor-netty-core/src/main/java/reactor/netty/transport/ClientTransport.java

```diff
@@ -140,6 +142,48 @@ public T doOnDisconnected(Consumer<? super Connection> doOnDisconnected) {
 		return dup;
 	}
 
+	/**
+	 * Set or add a callback called before {@link SocketAddress} is resolved.
+	 *
+	 * @param doOnResolve a consumer observing resolve events
+	 * @return a new {@link ClientTransport} reference
+	 */
+	public T doOnResolve(Consumer<? super Connection> doOnResolve) {
+		Objects.requireNonNull(doOnResolve, "doOnResolve");
+		T dup = duplicate();
+		Consumer<Connection> current = ((ClientTransportConfig)configuration()).doOnResolve;
+		dup.configuration().doOnResolve = current == null ? doOnResolve : current.andThen(doOnResolve);
+		return dup;
+	}
+
+	/**
+	 * Set or add a callback called after {@link SocketAddress} is resolved.
```

Comment:
nit: `is resolved successfully.` instead of `is resolved.`

Reply:
@anilgursel please explain how this conversation is resolved, nothing seems to have changed so at a minimum you should answer the comment rather than sneaky closing the conversation...

Reply:
@anilgursel @simonbasle I think javadoc update is added by mistake on the `doOnResolve` method above instead of `doAfterResolve` method

Reply:
@simonbasle Apparently, I added the comment on the wrong one.  Sorry for the mistake, and thank you for catching it!